### PR TITLE
feat: add new llm provider: Novita AI

### PR DESCRIPTION
**Pull request by jasonhp**
Add new model provider Novita AI.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (evilsocket) [inline .vscode/launch.json]: this is unnecessary

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ While Nerve was inspired by other projects such as Autogen and Rigging, its main
 
 ## LLM Support
 
-Nerve features integrations for any model accessible via the [ollama](https://github.com/ollama/ollama), [groq](https://groq.com), [OpenAI](https://openai.com/index/openai-api/), [Fireworks](https://fireworks.ai/) and [Huggingface](https://huggingface.co/blog/tgi-messages-api#using-inference-endpoints-with-openai-client-libraries) APIs. 
+Nerve features integrations for any model accessible via the [ollama](https://github.com/ollama/ollama), [groq](https://groq.com), [OpenAI](https://openai.com/index/openai-api/), [Fireworks](https://fireworks.ai/), [Huggingface](https://huggingface.co/blog/tgi-messages-api#using-inference-endpoints-with-openai-client-libraries) and [NovitaAI](https://novita.ai/model-api/product/llm-api?utm_source=github_nerve&utm_medium=github_readme&utm_campaign=link) APIs.
```
Comment (evilsocket) [inline README.md]: please remove utm tracking

File: crates/ollama-rs/src/generation/chat/mod.rs [hunk 3]
```diff
@@ -199,14 +199,13 @@ impl Ollama {
     /// Without impact for existing history
     /// Used to prepare history for request
     fn get_chat_messages_by_id(&mut self, history_id: impl ToString) -> Vec<ChatMessage> {
+        let mut new_history = std::sync::Arc::new(std::sync::RwLock::new(MessagesHistory::default()));
```
Comment (evilsocket) [inline crates/ollama-rs/src/generation/chat/mod.rs]: why this change?

**Review by evilsocket — CHANGES_REQUESTED** [review]

File: crates/ollama-rs/src/generation/chat/mod.rs (hunk 3 above)
Comment (jasonhp) [reply crates/ollama-rs/src/generation/chat/mod.rs]: Cargo build not successful for me until I do this change. It throws an Error: "creates a temporary value which is freed while still in use".

File: .vscode/launch.json (hunk 1 above)
Comment (jasonhp) [reply .vscode/launch.json]: File removed.

File: README.md (hunk 2 above)
Comment (jasonhp) [reply README.md]: Removed

File: crates/ollama-rs/src/generation/chat/mod.rs (hunk 3 above)
Comment (evilsocket) [reply crates/ollama-rs/src/generation/chat/mod.rs]: rustc --version?

Comment (jasonhp) [reply crates/ollama-rs/src/generation/chat/mod.rs]: My Rust version was old. Restored.
